html/css/graph.css

```diff
@@ -0,0 +1,265 @@
+/********************************************/
```

Comment:
Could you add some `<div>` that wraps all HTML content in `graph.pl` and add this as a parent selector to everything in here? Right now, some of this CSS overrides base Foundation/OFF styling, so that some styling outside the graph area is affected incorrectly:
![image](https://user-images.githubusercontent.com/87124/92322763-81ce4a00-f033-11ea-9456-1f2f91679bef.png)

Reply:
I added in graph.pl a div container with **class="graph"** and added this in front of styling in graph.css. However, the styling problem with "change your account parameters" is still present : it may not be introduced by **graph.css** styling but I noticed the **svg** class="icon" inside the **li** tag item vanished when one clicks on the "graph" button from the product display page. I could not find out the purpose of this behaviour.